diff --git a/node_modules/flowbite-svelte-icons/dist/StarOutline.svelte b/node_modules/flowbite-svelte-icons/dist/StarOutline.svelte
index 0f1cc59..c0beb4e 100644
--- a/node_modules/flowbite-svelte-icons/dist/StarOutline.svelte
+++ b/node_modules/flowbite-svelte-icons/dist/StarOutline.svelte
@@ -34,7 +34,7 @@
 	{color}
 	{...restProps}
 	class={cn('shrink-0', sizes[size], className)}
-	viewBox="0 0 24 24"
+	viewBox="0 1 24 24"
 	aria-label={ariaLabel}
 	aria-describedby={hasDescription ? ariaDescribedby : undefined}
 	aria-hidden={!isLabeled}
